scripts/apps/authoring/authoring/controllers/ChangeImageController.js

```diff
@@ -328,7 +343,9 @@ export function ChangeImageController($scope, gettext, notify, modal, _, api, $r
         }}).then((result) => {
             $scope.data.item.renditions = result.renditions;
             $scope.data.item._etag = result._etag;
-            $scope.data.metadata = $scope.data.item;
+            const editableMetadata = extractEditableMetadata($scope.data.metadata);
+
+            $scope.data.metadata = angular.extend($scope.data.item, editableMetadata);
```

Comment:
Is `angular.extend` different than native `Object.assign` or `const three = {...one, ...two}` ?

Reply:
yes, angular.extend changes first parameter and return it

Reply:
> yes, angular.extend changes first parameter and return it

That's how `Object.assign` works.

Reply:
I think angular.extend goes deep on merging and Object.assign is not going deep

Reply:
yes, actually deep merge is not needed because I update only text values
I will change to  Object.assign